### ISSUE TITLE
Bug: Profile display of participated opportunities

### ISSUE THREAD

**Issue by kbwagner** [openopps/openopps-platform#1870]
Steps to Reproduce:
1) Apply to at least one opportunity
2) Go to your profile
3) For participated opportunities, first column should say "Status" and status should be text only (not pill)
- Applied
- Assigned
- Not Assigned
- Complete

![image](https://user-images.githubusercontent.com/18709918/37115467-3551ebb8-2219-11e8-8ba1-80831fcf13cf.png)

From 3/6/18 Bug bash

**Comment by bmcorum** [comment]
@margaretdumas and @Kibrahacha67 this is ready for testing

**Comment by kbwagner** [comment]
3/19/18 - Per discussion with Min and Brice, "Your status" will now be "Status"Brice Corum [1:30 PM]
If the task is canceled should it show canceled?
Concern I have is with the table header “your status” since that would show when I view someone else’s profile

Min Chung [1:44 PM]
Ahhh I see
For Created opportunities, it should show canceled. For Participated, I don’t think we show
Maybe for Participated, we should just say ‘Status’ and not Your status
I hadn’t thought about when someone else is looking at your profile. ahhh

Brice Corum [1:54 PM]
For participated we do collect all task participated/applied for all states (open, not open, in progress, completed, canceled) (edited)
So I added the following logic:

if opportunity is completed and you are assigned 'Complete' else 'Not assigned'

if opportunity is in progress and you are marked task complete 'Complete' else if assigned then 'Assigned' else 'Not assigned'

if opportunity is cancled then 'Canceled'

default if assigned 'Assigned' else 'Applied' (edited)
Confuse you yet?

Min Chung [3:58 PM]
I got it

Min Chung [4:04 PM]
I was trying to go back and see what we wrote. That makes sense.
